core/runtime.rs

```diff
@@ -369,6 +369,13 @@ impl JsRuntime {
     state.js_recv_cb.replace(v8::Global::new(scope, cb));
   }
 
+  /// Ensures core.js has the latest op-name to op-id mappings
+  pub fn sync_ops_cache(&mut self) {
+    self
+      .execute("deno:core/internals", "Deno.core.syncOpsCache()")
```

Comment:
Maybe "<anon>" is a better name for this script.

Reply:
Done